static/app/gettingStartedDocs/javascript/remix.tsx

```diff
@@ -81,16 +81,20 @@ export const steps = ({
       },
       {
         language: 'javascript',
-        description: tct(
-          `Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [action] and [loader] functions. You can also initialize Sentry's database integrations, such as Prisma, to get spans for your database calls:`,
-          {
-            action: (
-              <ExternalLink href="https://remix.run/docs/en/v1/api/conventions#action" />
-            ),
-            loader: (
-              <ExternalLink href="https://remix.run/docs/en/1.18.1/api/conventions#loader" />
-            ),
-          }
+        description: (
+          <div>
```

Comment:
Should this div be here or rather in the render component
https://github.com/getsentry/sentry/blob/cb0cc4efc48dfb60689566df1d72506ec8a6a1c4/static/app/components/onboarding/gettingStartedDoc/layout.tsx#L34
so that this does not resurface again on other platforms?

Reply:
I'll double-check the layout component and make some necessary tweaks, but I'd rather merge this fix first if that's ok